### DISCUSSION TITLE
Ryven 3

### DISCUSSION THREAD

**Discussion by leon-thomm** [leon-thomm/Ryven#63] [General]
**Complete Ryven remake**
Over the past 6 monts I've been abstracting away underlying frameworks reimplementing the core functionality of the software which led to a complete remake of Ryven itself. Ryven 3 is a completely new software (no backwards compatibility to Ryven 2).

Most important change regarding Ryven is the new packages system. The NodeManager has been removed completely, the new nodes system is very different, much simpler and more scalable. Ryven 3 includes small exmaple projects and some default nodes. The new [website](https://ryven.org) now has a complete guide.

The underlying frameworks are `ryvencore` (backend) and `ryvencore-qt` (frontend), the repository is [here](https://github.com/leon-thomm/ryvencore-qt). `ryvencore` and `ryvencore-qt` are completely separated, which brings a whole lot of improvements, see [ryvencore-qt](https://github.com/leon-thomm/ryvencore-qt) and the [website](https://ryven.org) for more details.

<hr /><em>This discussion was created from the release <a href='https://github.com/leon-thomm/Ryven/releases/tag/v3.0.0-alpha'>Ryven 3</a>.</em>

**Comment by dumblob** [comment]
Congratulations! I've skimmed the new doc, the new page and of course the new ryvencore and ryvencore-qt and it looks pretty decent.

I think in this shape it's ready for pilots as a pre-step of production deployment. I'm certain customers will like it - let me know how it goes with AppInnovators!

**Comment by tom-cosmol** [comment]
I find it very interesting and would like to use it to solve a task. I have a question about this: I can do a lot in the example graphics, including creating and connecting nodes. However, I have not managed to simply delete connections again. Is this not finished and do I have to implement it myself?
